utils/das_data_sorter.py

```diff
@@ -1,6 +1,79 @@
 import pandas as pd 
 import argparse 
 from math import ceil
+from numpy import median, mean
+
+# averages data within the seconds or minutes, according to new_time array
+def avg_Data(array):
+    avg_array = []
+
+    for millisecond_indices in new_time:
+        avg = mean(array[millisecond_indices])
+        avg = round(avg, ndigits=2)
+        avg_array.append(avg)
+
+    return avg_array
+
+# 3 point moving mean smoothing
+def mean_smooth(data):
+    new_data = []
+
+    for i in range(1,len(data)-1):
+        first = data[i-1]
+        second = data[i]
+        third = data[i+1]
+
+        new_data_point = mean([first, second, third])
+        new_data_point = round(new_data_point, ndigits=2)
```

Comment:
I'd like to control the number of digits to round. Can you add a new `precision=2` argument to the `mean_smooth` function (By adding `=2`, we set a default parameter if the user does not include a second parameter to this function.